rootless-install.sh

```diff
@@ -225,10 +225,13 @@ exec_setuptool() {
 }
 
 do_install() {
+	echo "# Executing docker rootless install script, commit: $SCRIPT_COMMIT_SHA"
+
 	init_vars
 	checks
 
 	tmp=$(mktemp -d)
+	# shellcheck disable=SC2064
 	trap "rm -rf $tmp" EXIT INT TERM
```

Comment:
This is close enough to the `EXIT` event within this script that this could instead be something like this without much risk:

```suggestion
	trap 'rm -rf "$tmp"' EXIT INT TERM
```

Would that be enough to avoid having to tell off shellcheck?

Reply:
looks like it's happy with that (at least in my IDE); thanks! I wasn't 100% sure how to fix with the right quoting, so took the easy approach.